backend/dataall/modules/datasets/indexers/table_indexer.py

```diff
@@ -1,4 +1,5 @@
 """Indexes DatasetTable in OpenSearch"""
+
```

Comment:
ruff linting change (already in https://github.com/data-dot-all/dataall/pull/1137 )